mppsolar/protocols/abstractprotocol.py

```diff
@@ -2,6 +2,7 @@
 import calendar  # noqa: F401
 import logging
 import re
+from datetime import datetime
```

Comment:
is this needed?

Reply:
Needed to decode inverter date and time QT command